awx/ui/client/src/access/rbac-role-column/roleList.directive.js

```diff
@@ -21,9 +21,22 @@ export default
                         }))
                         .concat(scope.deleteTarget.summary_fields
                             .indirect_access.map((i) => {
-                                i.role.explicit = false;
-                                return i.role;
+                                if (i.descendant_roles.includes('admin_role')) {
+                                    i.role.explicit = false;
+                                    i.role.name = 'Admin';
+                                    return i.role;
+                                } else {
+                                    let indirect_roles = [];
+                                    i.descendant_roles.forEach((descendant_role) => {
+                                        let r = JSON.parse(JSON.stringify(i.role));
```

Comment:
this seems to work just fine. That said, if `angular.copy` or `_.cloneDeep` work, it might be wise to use one of them instead.

Reply:
:+1: